adoc/admin-storage-vsphere.adoc

```diff
@@ -0,0 +1,248 @@
+= vSphere Storage
+The vSphere cloud provider can be enabled with {productname} to allow Kubernetes pods to use VMWare vSphere Virtual Machine Disk (VMDK) volumes as persistent storage.
+
+This chapter provides the two types of persistent volume usage and description.
+
+== Node Meta
+Extra node meta could be found when region and zone was added to `vsphere.conf` before bootstrap cluster node.
+
+====
+    [Labels]
+    region = "<VC_DATACENTER_TAG>"
+    zone = "<VC_CLUSTER_TAG>"
+====
+
+You can view the cloudprovider associated node meta with command.
+----
+kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name}{"\tregion: "}{.metadata.labels.failure-domain\.beta\.kubernetes\.io/region}{"\tzone: "}{.metadata.labels.failure-domain\.beta\.kubernetes\.io/zone}{"\n"}{end}'
+...
+010084072206    region: vcp-provo       zone: vcp-cluster-jazz
+010084073045    region: vcp-provo       zone: vcp-cluster-jazz
+----
```

Comment:
I'm a little confused about what this section what to achieve, at `adoc/deployment-bootstrap.adoc` we have described the region & zone, so I think the section would be suitable change to `adoc/deployment-bootstrap.adoc`, WDYT?

Reply:
Yes is relate the the zone and region setting mentioned in `adoc/deployment-bootstrap.adoc`. I'v kept it separate to talk about node meta.

Reply:
Okay, I realized more on this, the region and zone metadata have to be manually created to vSphere cluster.
If the user wants to use VMDK as storage, the user has to specify the region and zone in `vsphere.conf` before bootstrapping the node.

Reply:
Maybe it deserves adding some relationship between volume and zone? (ref: https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/zones.html#zone-for-volume.)

To @jenting, iirc the zone and region labelings are not necessary for volume provision but it is more about info here.

Reply:
Enhanced zone and region info.

Reply:
Good job. Thanks.